.env.example

```diff
@@ -1,2 +1,2 @@
-YII_ENV=
+YII_ENV=/
```

Comment:
?

Reply:
[This is default name of environment](https://github.com/yiisoft/config/blob/master/src/Options.php#L20)

Reply:
Set `dev` by default